docs/source/for_users/modules/training_unccdreporting.rst

```diff
@@ -63,6 +63,54 @@ in the menu if the **Download remotely-generated datasets automatically** is che
    :align: center
 
 7. The default data: Soil organic carbon degradation, land cover degradation, land productivity dynamics, and population dissaggregated by gender, will be added to the map for the region of interest.
+
+
+Compute SDG 15.3.1 Indicator and UNCCD's SO 2-3
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- **Objective**: Learn to integrate the land cover, primary productivity and soil organic carbon sub-indicators to compute SDG 15.3.1 Indicators in raster format and tabular outputs with areas estimated.
+You will also learn to estimate the proportion of population exposed to land degradation, disaggregated by gender. 
```

Comment:
This needs to be one line - list can't split across a line without proper indentation (or, don't use a linebreak like we have here)